### ISSUE TITLE
react strict mode produces tons of errors

### ISSUE THREAD

**Issue by pdeva** [ant-design/ant-design#15193]
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[http://notfound.com](http://notfound.com)

### Steps to reproduce
use a table component

### What is expected?
React.strictmode shouldnt produce any errors 

### What is actually happening?
See errors like this in the console:
https://www.screencast.com/t/9Asek8erO

| Environment | Info |
|---|---|
| antd | 3.14.1 |
| React | 16.8 |
| System | windows |
| Browser | chrome |

**Comment by afc163** [comment]
Right, due to short of hands, could you help us to resolve errors which like #9792 or `findDOMNode` warnings, before react@17 comes?

**Comment by afc163** [comment]
Merge into #9792

**Comment by alissaVrk** [comment]
was this fixed? I can still see the warnings
